README.md

```diff
@@ -132,13 +132,13 @@ return a new object, leaving the original object unaffected:
 $ten = BigInteger::of(10);
 
 echo $ten->plus(5); // 15
-echo $ten->multipliedBy(3); // 30
+echo $ten->multipliedBy(3); // 45
```

Comment:
You have a point, though, I could maybe choose different numbers for the two examples, as indeed it could make one think that I mean to demonstrate equivalence!